allensdk/brain_observatory/behavior/behavior_project_api/behavior_ophys_analysis_query_utils.py

```diff
@@ -0,0 +1,309 @@
+from pymongo import MongoClient
+import yaml
+import pandas as pd
+import numpy as np
+import json
+import os
+import glob
+import traceback
+import datetime
+
+import time
+from tqdm import tqdm as progress
+
+from IPython.core.display import display, HTML
```

Comment:
probably don't want this?

Reply:
Correct. Artifact from development in jupyter!